platform/z1/dev/tmp102.c

```diff
@@ -43,6 +43,12 @@
 #include "i2cmaster.h"
 #include "tmp102.h"
 
+#ifdef TMP_DEBUG
+#define PRINTFDEBUG(...) printf(__VA_ARGS__)
+#else
+#define PRINTFDEBUG(...)
+#endif
+
```

Comment:
isn't `PRINTFDEBUG` defined in: https://github.com/contiki-os/contiki/blob/master/platform/z1/dev/i2cmaster.h ?